packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.tsx

```diff
@@ -0,0 +1,93 @@
+// Components
+import VExpansionPanelHeader, { makeVExpansionPanelHeaderProps } from './VExpansionPanelHeader'
+import VExpansionPanelContent from './VExpansionPanelContent'
+import { VExpansionPanelSymbol } from './VExpansionPanels'
+
+// Composables
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeGroupItemProps, useGroupItem } from '@/composables/group'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { useBackgroundColor } from '@/composables/color'
+import { makeTagProps } from '@/composables/tag'
+import { makeLazyProps } from '@/composables/lazy'
+
+// Utilities
+import { computed, provide } from 'vue'
+import { defineComponent } from '@/util'
+
+export default defineComponent({
+  name: 'VExpansionPanel',
+
+  props: {
+    header: String,
+    content: String,
```

Comment:
should we rename this to **text** to match other components? If not, what is the difference between the two?

Reply:
See comment on `header` name